copilot/fastapi/tests/test_tvp.py

```
@@ -0,0 +1,141 @@
+        app_dir.name = "test-app"
+        app_dir.is_dir.return_value = True
+        
+        values_file = MagicMock()
+        values_file.exists.return_value = True
+        
+        # Setup directory structure for iterdir calls
+        namespace_dir.iterdir.return_value = [app_dir]
+        mock_iterdir.return_value = [namespace_dir]
+        
+        # Setup Path/file for values.yaml using a custom __truediv__ implementation
+        def mock_truediv(self, other):
```

Ruff (0.8.2)

Missing return type annotation for private function `mock_truediv`

(ANN202)